profile-widget/diveeventitem.cpp

```diff
@@ -81,19 +81,19 @@ void DiveEventItem::setupPixmap()
 #define EVENT_PIXMAP(PIX) QPixmap(QString(PIX)).scaled(sz_pix, sz_pix, Qt::KeepAspectRatio, Qt::SmoothTransformation)
 #define EVENT_PIXMAP_BIGGER(PIX) QPixmap(QString(PIX)).scaled(sz_bigger, sz_bigger, Qt::KeepAspectRatio, Qt::SmoothTransformation)
 	if (same_string(internalEvent->name, "")) {
-		setPixmap(EVENT_PIXMAP(":warning-icon"));
+		setPixmap(EVENT_PIXMAP(":status-warning-icon"));
 	} else if (internalEvent->type == SAMPLE_EVENT_BOOKMARK) {
-		setPixmap(EVENT_PIXMAP(":flag"));
+		setPixmap(EVENT_PIXMAP(":bookmark-icon"));
```

Comment:
this isn't a regular bookmark. This is really a dive flag. Not sure about this one.

Reply:
You are right. I will change this to more appropriate *dive-bookmark-icon*.